### PR TITLE
Fix release build notes formatting

### PR DESCRIPTION
**Pull request by ehsannas**
Fix the table formatting for release_build_files/readme.

See [original](https://github.com/firebase/firebase-cpp-sdk/blob/main/release_build_files/readme.md) vs. [new](https://github.com/ehsannas/firebase-cpp-sdk/blob/ehsannas-patch-1/release_build_files/readme.md).

### CODE REVIEW

**Review by vimanyu — COMMENTED** [review]
Thanks for catching this and fixing it. 
Could you change the PR title to a more descriptive one, maybe "Fix release build notes formatting"?
LGTM otherwise. Will approve once the merge conflict is resolved.

**Comment by ehsannas** [comment]
@vimanyu Ah sorry didn't notice the title. It was decided automatically by the GitHub UI 😄  PTAL

**Review by vimanyu — APPROVED** [review]
LGTM

**Pull request merged**
